sdks/java/core/src/main/java/org/apache/beam/sdk/testing/WindowFnTestUtils.java

```diff
@@ -67,14 +68,30 @@
   public static <T, W extends BoundedWindow> Map<W, Set<String>> runWindowFn(
       WindowFn<T, W> windowFn,
       List<Long> timestamps) throws Exception {
+    // lift List<Timestamp> into List<TimestampedValue> to factorize implementation
+    List<TimestampedValue<T>> timestampedValues = new ArrayList<>();
+    for (Long timestamp : timestamps){
+      timestampedValues.add(TimestampedValue.of((T) null, new Instant(timestamp)));
+    }
+    return runWindowFnWithValue(windowFn, timestampedValues);
+
+  }
+
+  /**
+   * Runs the {@link WindowFn} over the provided input, returning a map
+   * of windows to the timestamps in those windows. This version allows to pass a list of
```

Comment:
Unrelated, but odd, and maybe worth addressing. This says it is a map from windows to the timestamps in those windows, but it is a `Set<String>`. Should this be `Set<Instant>`?